src/visualizers/panels/ArtifactIndex/ArtifactIndexControl.js

```diff
@@ -72,12 +72,11 @@ define([
         };
         this._widget.getConfigDialog = () => new ConfigDialog(this._client);
 
-        this._widget.onNameChange = (id, newName) => {
-            var name = this._client.getNode(id).getAttribute('name'),
-                msg = `Renamed "${name}" artifact to "${newName}"`;
-
+        this._widget.onAttributeChange = (id, attr, newValue) => {
+            const attrValue = this._client.getNode(id).getAttribute(attr),
+                msg = `Changed "${attr}'s" value from ${attrValue}" to "${newValue}"`;
```

Comment:
It might be nice to have a more meaningful commit message. Specifically, this isn't clear what artifact was being edited. This wasn't an issue when we were editing the name since that is how people normally identify these :)

Reply:
Ok. Should this artifact's id be included in the commit message as well? Something like 
```javascript
msg = `Changed attribute "${attr}'s" value  for node with id : "${id}" from "${attrValue}" to "${newValue}"`;
```

Reply:
No need as it is already there:

![DeepinScreenshot_select-area_20200507122231](https://user-images.githubusercontent.com/4982789/81325013-617a8880-905d-11ea-883d-e736ce97d3c4.png)

Reply:
I think it is most important that it is understandable to the end users at a glance, so I would recommend something like `Set ${name} artifact's ${attr} to ${newValue} (from ${oldValue})`